app/gcl/agwa/section6/page.css

```
@@ -0,0 +1,568 @@
+    padding: 20px 24px;
+    color: var(--text-dim);
+    font-size: 14.5px;
+    line-height: 2;
+}
+
+.agwa-section6-root .main a {
+    color: var(--accent);
+    text-decoration: none;
+    border-bottom: 1px solid var(--accent-soft);
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)